### PR TITLE
[15.0][MIG] event_track_location_overlap: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza can you review it?

@Tecnativa TT41012

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested :+1: Thanks!

File: event_track_location_overlap/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Jairo Llopis
+  * Cristina Martin R.
```
Comment (pedrobaeza) [inline event_track_location_overlap/readme/CONTRIBUTORS.rst:4]: Add yourself to contributors

File: event_track_location_overlap/tests/test_overlap.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                    "date": sample.b_start,
+                    "duration": sample.b_duration,
+                    "event_id": self.event.id,
+                    "location_id": self.location.id,
+                    "name": "test track b",
+                }
+            )
+            if raise_always:
+                # This notifies good track creation but rolls it back
+                raise Warning(_("%(sample)s worked!") % {"sample": sample})
```
Comment (pedrobaeza) [inline event_track_location_overlap/tests/test_overlap.py]: There's no sense in using Odoo translatable exceptions, as this are tests. Use a normal exception.

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**